andela/trime
# Show a message if the current project has no time tracks logged yet.

### What does this PR do?
Show a message if the current project has no time tracks logged yet.

### Description of Task to be completed?
Add a message to show that the user hasn't started tracking time yet. If the user starts tracking time, the message should be updated to show that they have just started tracking time.

### How should this be manually tested?
Create a new project and click on that project. Before you start tracking time, you should see a message informing you that you haven't started tracking time yet.

<img width="1680" alt="screen shot 2016-06-28 at 12 35 34" src="https://cloud.githubusercontent.com/assets/17286899/16410895/3ed6f908-3d2d-11e6-8fcb-b07d29723749.png">


Start tracking time and this message should change to inform you that you are now tracking time.

<img width="1680" alt="screen shot 2016-06-28 at 12 35 54" src="https://cloud.githubusercontent.com/assets/17286899/16410901/47e67d2a-3d2d-11e6-8d7b-def4d49ef9b5.png">


If you pause or stop the time track, the message should disappear and display the tracked time instead.

<img width="1680" alt="screen shot 2016-06-28 at 12 36 07" src="https://cloud.githubusercontent.com/assets/17286899/16410907/4de8f2e8-3d2d-11e6-86c5-80d9b3360ce1.png">


### What are the relevant pivotal tracker stories?
[Show a message if the current project has no time tracks logged yet.](https://www.pivotaltracker.com/story/show/120096931)
